qtpy/QtGui.py

```diff
@@ -8,7 +8,30 @@
 
 """Provides QtGui classes and functions."""
 
-from . import PYQT6, PYQT5, PYSIDE2, PYSIDE6
+from . import PYQT6, PYQT5, PYSIDE2, PYSIDE6, QtModuleNotInstalledError
+from .utils import _getattr_missing_optional_dep
```

Comment:
```suggestion
from .utils import getattr_missing_optional_dep
```

I don't think this should be a private function because it's imported and used in other places in our code (like here).

Reply:
Well, it's private to the _package_ as a whole rather than the specific module its defined in. At present, this means it is exposed as a module-level public function in any of the modules its imported in.